app.js

```diff
@@ -175,12 +175,12 @@ if (
 try {
 	lastCommit = git.getLastCommit();
 } catch (err) {
-	logger.debug('Cannot get last git commit', err.message);
+	appLogger.debug('Cannot get last git commit', err.message);
```

Comment:
this isn't convincing, we are catching an error and logging it as `debug`. I agree it was like that before, but good to change it now.

Reply:
I think this `debug` level is intention. Especially for built package, i dont think there will be a git included.